scholly/base/src/components/publications/PublicationPage.js

```diff
@@ -10,49 +10,45 @@ import Container from 'react-bootstrap/Container';
 import Publications from './Publications';
 // import { TEAM_INFO } from '../../global/data';
 
-const PublicationPage = () => {
-  // const linkedHandle = TEAM_INFO.twitterHandle;
-
-  return (
-    //
-    // Logic of showing twitter gadget. Removed for now. Wait for a better design.
-    //
-    // <Fragment>
-    //   {linkedHandle ? (
-    //     <Container fluid className="pages-top-padding ">
-    //       <Row>
-    //         <Col className="pub-main-left">
-    //           <Container className="text-center mt-3 mb-3">
-    //             <div className="publication-pg-title">Our Publications</div>
-    //           </Container>
-    //           <Publications />
-    //         </Col>
-    //         <Col className="pub-twitter-right">
-    //           <TwitterFeed linkedHandle={linkedHandle} />
-    //         </Col>
-    //       </Row>
-    //     </Container>
-    //   ) : (
-    //     <Fragment>
-    //       <Container className="pages-top-padding text-center mt-3 mb-3">
-    //         <div className="publication-pg-title">Our Publications</div>
-    //       </Container>
-    //       <Container fluid>
-    //         <Publications />
-    //       </Container>
-    //     </Fragment>
-    //   )}
-    // </Fragment>
-
-    <Fragment>
-      <Container className="pages-top-padding text-center mt-3 mb-3">
-        <div className="publication-pg-title">Our Publications</div>
-      </Container>
-      <Container fluid>
-        <Publications />
-      </Container>
-    </Fragment>
-  );
-};
+const PublicationPage = () => (
+// const linkedHandle = TEAM_INFO.twitterHandle;
+  //
+  // Logic of showing twitter gadget. Removed for now. Wait for a better design.
+  //
+  // <Fragment>
+  //   {linkedHandle ? (
+  //     <Container fluid className="pages-top-padding ">
+  //       <Row>
+  //         <Col className="pub-main-left">
+  //           <Container className="text-center mt-3 mb-3">
+  //             <div className="publication-pg-title">Our Publications</div>
+  //           </Container>
+  //           <Publications />
+  //         </Col>
+  //         <Col className="pub-twitter-right">
+  //           <TwitterFeed linkedHandle={linkedHandle} />
+  //         </Col>
+  //       </Row>
+  //     </Container>
+  //   ) : (
+  //     <Fragment>
+  //       <Container className="pages-top-padding text-center mt-3 mb-3">
+  //         <div className="publication-pg-title">Our Publications</div>
+  //       </Container>
+  //       <Container fluid>
+  //         <Publications />
+  //       </Container>
+  //     </Fragment>
+  //   )}
+  // </Fragment>
```

Comment:
Can this commented section be removed?

Reply:
This is still here because Jie plans on adding the twitter box to the deployed website, at the time he couldn't figure out a way to make it look nice so it's been commented out. iirc there's plans to use this code later so I'm leaving this as is.

Reply:
I will change it asap (probably this weekend) if we're still using this layout without much amendment.